CHANGES.md

```diff
@@ -1,3 +1,8 @@
+## 3.2.1 - February 15, 2018
+Updates:
+-   Updated PBAC to version 0.2.0 (lodash vulnerability update)
+-   Updated iam.js (StringLike param order reversed)
+
```

Comment:
you should put these under 3.2.0 as we did not release a new version of the module

Reply:
added under 3.2.0 comments